### ISSUE TITLE
A Set of tools for each Track II Team

### ISSUE THREAD

**Issue by timmwille** [ASKnet-Open-Training/Minimal-Repair-Setup#2]
@dawaedina27 @yyenki @maliarich @romeoplat 

→ summen the teams :) → I think it is time to talk about tools :)

**Comment by timmwille** [comment]
Berlin Germany might go with this:
![CommunityRepair-Example2-3](https://user-images.githubusercontent.com/8081055/105654717-e002a900-5ebe-11eb-8690-ef2a9ad63af7.jpg)

**Comment by timmwille** [comment]
@chardso @vugawilliam → any thoughts on it? what is missing/can be improved?

**Comment by vugawilliam** [comment]
Thank, I will update later today evening about the missing tools and
materials.

Thank

**Comment by dawaedina27** [comment]
cell phone tools are included in this set right

**Comment by timmwille** [comment]
> cell phone tools are included in this set right

yes @dawaedina27 they are, but then again, it is just my setup, I will share the exact list, but you need to find your own setup in the market → try to estimate the costs and we will make it happen. Also think about a casing for all the tools, could be a small backpack or a shoulder bag? → I'm curious what you guys come up with :)

**Comment by maliarich** [comment]
> cell phone tools are included in this set r
Very good concern @dawaedina27 .

**Comment by maliarich** [comment]
The tool kit is absolutely perfect! For bigger devices excluding most modern phones.

**Comment by chardso** [comment]
In the #Askotek these are missing,@vuga you will add on 
-  CR 2032/3v batteries.
-   9v batteries.
-   Mobile phone repair kit
-   Small combination padlock.
-   Diodes.
-   Disinfection spray.

**Comment by timmwille** [comment]
> The tool kit is absolutely perfect! For bigger devices excluding most modern phones.

modern phones have these small star screwdrivers, should work with the extra little tool bag in the right corner?

**Comment by timmwille** [comment]
> In the #Askotek these are missing,@vuga you will add on
> 
>     * CR 2032/3v batteries.
> 
>     * 9v batteries.
> 
>     * Mobile phone repair kit
> 
>     * Small combination padlock.
> 
>     * Diodes.
> 
>     * Disinfection spray.

you refer to the #ASKotec in YEF? 
@bmen check this out maybe at some point we can help with the stockroom

**Comment by chardso** [comment]
I refer #ASKotek YEF.

**Comment by vugawilliam** [comment]
I would real wish us to think about the how to improve ASKotec kit! This time and documenting of our activities that can be shareable online.

**Comment by timmwille** [comment]
Hey guys @chardso @PyGamma @vugawilliam @gibertgibs @dawaedina27 @romeoplat @B-yine @yyenki @Biorology @babyomer @wunbaai @yarmanyang @maliarich @ASKnet-Open-Training/track-ii-2020 please be aware of documenting your individual Repair Setup Tool Kit + Materials after shopping them :) and dont forget to think about a toolbag to organise and carry all those tools (it is a small extra kit additional to the #ASKotec should not get mixed up)

**Comment by chardso** [comment]
Am  set just waiting for a call to shopping.

**Comment by timmwille** [comment]
@chardso you got the call now? :) Adjumani has its Repair Cafe at the 7th of March even from what I understood @vugawilliam :) keep me in the loop @yyenki also @elobijo and @Biorology we check tomorrow

**Comment by chardso** [comment]
Tool kit for #6 
![Uploading IMG_20210311_153610.jpg…]()

**Comment by maliarich** [comment]
Eden Repair Cafe Tool Kit.
![Screenshot_20210313-085547.png](https://user-images.githubusercontent.com/56769901/111020995-8c9ada80-83da-11eb-99b2-d619dc6a3af3.png)

**Comment by chardso** [comment]
Adjumani team toolkit ,consisting of 63pieces .
![20210313_103057-1](https://user-images.githubusercontent.com/71114711/111024226-87e02180-83ee-11eb-878c-f9a78b3699a5.jpg)

**Comment by maliarich** [comment]
Hi @babyomer , @elobijo 
Can you please upload the tool kit (list) here
Next option;- send me the tools lists + pic,  I'll upload.

**Comment by maliarich** [comment]
> Adjumani team toolkit ,consisting of 63pieces .
> ![20210313_103057-1](https://user-images.githubusercontent.com/71114711/111024226-87e02180-83ee-11eb-878c-f9a78b3699a5.jpg)
> 

@chardso  I think you can now complete your documentation since you have started 😊. List the 63 tools in the kit.

**Comment by chardso** [comment]
![Uploading 20210313_104722.jpg…]()

**Comment by maliarich** [comment]
Thank you @chardso .

**Comment by timmwille** [comment]
also check #3 #4 #5 #6 #7 #8 #9 for additional tool kit documentation
